apps/web/components/seo/head-seo.tsx

```diff
@@ -72,12 +73,11 @@ const buildSeoMeta = (pageProps: {
 
 export const HeadSeo = (props: HeadSeoProps): JSX.Element => {
   const defaultUrl = getBrowserInfo()?.url;
-  const image = getSeoImage("default");
 
   const { title, description, siteName, canonical = defaultUrl, nextSeoProps = {}, app, meeting } = props;
 
-  const truncatedDescription = truncate(description, 24);
```

Comment:
No need for a longer and shorter description anymore, everything should be long from now on. Looks better, and google is still fine with this length.